File: Project_Go_bigint/PrimeFactorization_bigint/systeminformation/systeminformation.go
package systeminformation

import (
	"fmt"
	"runtime"
	"time"
)

// PrintSystemInformation ...
func PrintSystemInformation() {
	t := time.Now()
	const layout = "2006/01/02 15:04:05"

	fmt.Println("=====================================================================================")
	fmt.Printf("%-10s : %s\n", "Date", t.Format(layout))
	fmt.Printf("%-10s : %s\n", "Language", "Go")
	fmt.Printf("%-10s : %s\n", "OS", runtime.GOOS)
	fmt.Println("=====================================================================================")
}

File: Project_Python/PrimeFactorization/SystemInformation/SystemInformation.py
# coding: utf-8

import datetime
import platform
from . import TimeFormatter

def printSystemInformation():
    print("=====================================================================================\n"
        "{0:10s} : {1}\n"
        "{2:10s} : {3}\n"
        "{4:10s} : {5}\n"
        "====================================================================================="
        .format(
            "Date", "{0:%Y-%m-%d %H:%M:%S}".format(datetime.datetime.now()),
            "OS", platform.system(),
            "Language", "Python3"
        ))
